src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -1,5 +1,7 @@
 package nl.tudelft.pixelperfect;
 
+import java.io.IOException;
+
```

Comment:
Tried it again, also on my laptop, it keeps indicating an error with a clean maven site each time.  I'll approve this for now, it seems to be on my end, since both you and Floris are not getting it.